tests/vasp/test_files.py

```diff
@@ -9,11 +9,14 @@
         ({"additional_vasp_files": ("CHGCAR",)}, ("POSCAR", "INCAR", "CHGCAR")),
     ],
 )
-def test_copy_vasp_outputs_static(vasp_test_dir, tmp_dir, copy_kwargs, files):
+def test_copy_vasp_outputs_static(
+    vasp_test_dir, tmp_path, monkeypatch, copy_kwargs, files
+):
     from pathlib import Path
 
     from atomate2.vasp.files import copy_vasp_outputs
 
+    monkeypatch.chdir(tmp_path)
```

Comment:
Ah so this fixture doesn't automatically cd into the directory? To be honest, this seems like my implementation is actually better as it saves boilerplate?

Reply:
I think it's often a good thing to be explicit about changing a test function's context. Since it's just 1 line, maybe acceptable amount of boilerplate? But feel free to close.